auth/cloud/common.go

```diff
@@ -0,0 +1,23 @@
+package cloud
+
+import "github.com/banzaicloud/banzai-types/constants"
+
+// Verifier validates cloud credentials
+type Verifier interface {
+	VerifySecret() error
+}
+
+// NewVerifier create new instance which implements `Verifier` interface
+func NewVerifier(cloudType string, values map[string]string) Verifier {
+	switch cloudType {
```

Comment:
Just a minor use `default:` to return nil:) in the end

Reply:
thx 👍 done :)